src/app/personside/visittkort/body/vergemal/Vergemal.tsx

```diff
@@ -27,6 +28,8 @@ const Vergeinformasjon = styled.div`
 
 function Verge(props: { verge: VergeInterface }) {
     const { verge } = props;
+    loggInfo('Vergemål omfang er ' + verge.omfang + ' og vergesakstype er ' + verge.vergesakstype);
```

Comment:
husk å kun logge dersom vergemål er `undefined` for å kunne fange opp typer som er glemt i mappern. trenger ikke logge alle vergemål